modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/pages/Project/Cases/CaseOutlet.tsx

```diff
@@ -59,7 +59,7 @@ const CaseOutlet = () => {
 	});
 
 	useEffect(() => {
-		setHeaderActions({actions, item: testrayCase, mutate});
+		setHeaderActions({actions: [], item: testrayCase, mutate});
```

Comment:
It would be better that you make this validation in a form component.
```suggestion
		setHeaderActions({actions, item: testrayCase, mutate});
```